src/e2e/puppeteer/__tests__/color.ts

```diff
@@ -121,3 +106,23 @@ it('Clear the text color when setting background color', async () => {
   expect(extractStyleProperty(cursorText!)?.backgroundColor).toBe('rgb(170, 128, 255)')
   expect(extractStyleProperty(cursorText!)?.color).toBe('#000000')
 })
+
+it('Empty <font> or <span> element will be removed after setting color to default.', async () => {
```

Comment:
Should just say `Empty <span>` since this test doesn't actually test empty `<font>`.

Reply:
I got it. I will split this test to `background color test` and `text color test`.